src/models/sender.ts

```diff
@@ -0,0 +1,5 @@
+export interface Sender {
+    stopID: string;
```

Comment:
In our documentation we write id in camelCase stopId, should we change the syntax?